test-data/unit/check-flags.test

```diff
@@ -479,12 +479,13 @@ main:5: error: Constraint 2 becomes List[Any] due to an unfollowed import
 
 [case testDisallowImplicitAnyNewType]
 # flags: --ignore-missing-imports --disallow-any=unimported
-from typing import NewType
+from typing import NewType, List
 from missing import Unchecked
 
-Baz = NewType('Baz', Unchecked)
-[out]
-main:5: error: Argument 2 to NewType(...) must be subclassable (got Any)
+# this case is handled elsewhere: Baz = NewType('Baz', Unchecked)
```

Comment:
Hm, yeah, that makes sense. I was just worried that if someone changes the wording, they have to change the tests in several places vs in one spot. But I guess it's not too much work.